script/Base.s.sol

```diff
@@ -64,9 +64,9 @@ contract BaseScript is Script, Sphinx {
         // Populate the segment and tranche count map.
         populateSegmentAndTranchCountMap();
 
-        // Load the maximum segment and tranche count for the current chain id.
-        maxSegmentCount = segmentCountMap[block.chainid];
-        maxTrancheCount = trancheCountMap[block.chainid];
+        // Load the maximum segment and tranche count for the current chain id. Set a default value if not found.
+        maxSegmentCount = segmentCountMap[block.chainid] == 0 ? 500 : segmentCountMap[block.chainid];
+        maxTrancheCount = trancheCountMap[block.chainid] == 0 ? 500 : trancheCountMap[block.chainid];
```

Comment:
I bet @PaulRBerg won't like ternary operators

Reply:
You mean like this

```solidity
if (segmentCountMap[block.chainid] == 0) {
    maxSegmentCount = DEFAULT_MAX_COUNT;
} else {
    maxSegmentCount = segmentCountMap[block.chainid];
}

if (trancheCountMap[block.chainid] == 0) {
    maxTrancheCount = DEFAULT_MAX_COUNT;
} else {
    maxTrancheCount = trancheCountMap[block.chainid];
}
```